scripts/configurator/parser/cmake_var.py

```diff
@@ -1,9 +1,92 @@
-from typing import Dict, Any, Type, Mapping
+from typing import Dict, Any, Optional, Type, Mapping, List, Callable
+import re
+from sys import stderr
+
+
+class metadata:
+   keyword_expr = re.compile(r"^([A-Z_]+) (.*)")
+
+   def __init__(self, comment: List[str]) -> None:
+      self.depends_on: Optional[List[str]] = None
+      self.group: Optional[str] = None
+      self.realtype: Optional[str] = None
+
+      self.kw_to_setter: Mapping[str, Callable]= {
+         "REALTYPE": self.get_setter_realtype(),
+         "DEPENDS_ON": self.get_setter_depends_on(),
+         "GROUP": self.get_setter_group(),
+      }
+
+      self.kw_to_var: Mapping[str, Any] = {
```

Comment:
I don't understand why you dropped the `parser` dict. You should have replaced ALL the dicts with one and put the values in a namedTuple. Not just dropping the parser dict.